kickstart/presentation.yaml

```
@@ -1,6 +1,6 @@
-summary: Create kickstart presentation
+summary: Prepare a kickstart presentation
 
-priority: major
+priority: should
```

To have a kickstart, we need a presentation.

Arguably, it doesn't need to be a "presentation", but I think this implicitly also covers what content should be in kickstart.

```suggestion
priority: must
```